### DISCUSSION TITLE
老师什么时候讲过RLHF？

### DISCUSSION THREAD

**Discussion by YuboFeng2023** [mli/paper-reading#256] [Q&A]
请问一下，老师团队什么时候讲过RLHF？

【InstructGPT 论文精读【论文精读·48】】 【精准空降到 21:25】 https://www.bilibili.com/video/BV1hd4y187CR/?share_source=copy_web&vd_source=609e9519945dd913bbd7a14b40d7508b&t=1285

上述链接指示的视频位置里面提到，“我们之前讲过RLHF方法”。

所以我想请问一下，老师什么时候讲过 RLHF 方法？

**Comment by crystalxian** [comment]
李沐老师在【21:25】左右提到的应该是RLHF是OPENAI团队之前的工作。RLHF的训练方法李沐老师在这个视频后面讲到了。小伙伴可以往后看下。
